packages/mediacenter/kodi/package.mk

```diff
@@ -19,6 +19,17 @@ configure_package() {
     PKG_KODI_USE_LTO="-DUSE_LTO=${CONCURRENCY_MAKE_LEVEL}"
   fi
 
+  # Set Linker matching DEFAULT_LINKER
+  if [ "${DEFAULT_LINKER}" = "gold" ]; then
+    PKG_KODI_LINKER="-DENABLE_GOLD=ON"
+  elif [ "${DEFAULT_LINKER}" = "mold" ]; then
```

Comment:
`DENABLE_GOLD` is correct? Deleted line below is `DENABLE_LDGOLD`.

Reply:
the flags changed upstream, current master package has broken gold linking